precompiles/slash/methods.go

```diff
@@ -15,7 +15,7 @@ const (
 	MethodSlash = "submitSlash"
 )
 
-// SubmitSlash Slash assets to the staker, that will change the state in slash module.
+// SubmitSlash NoInstantaneousSlash assets to the staker, that will change the state in slash module.
```

Comment:
Why this edit?

Reply:
It's a mistake, will change it in the next PR.